### DISCUSSION TITLE
debug配置问题

### DISCUSSION THREAD

**Discussion by LongZhouuu** [Xilesun/CS50x#11] [Q&A]
### 你的运行环境是?

MacOS

### 请详细描述你的问题，操作步骤。并附上完整的报错信息截图，或其他有效信息。

老师您好，我按照你的debug那篇视频配置debug，运行的时候显示doesn't exist
<img width="1156" alt="截屏2024-03-26 12 16 03" src="https://github.com/Xilesun/CS50x/assets/117184733/3b8f75bc-188e-45c5-9825-b2e6fc3be081">

这是我按照视频配置的launch
<img width="1156" alt="截屏2024-03-26 12 17 17" src="https://github.com/Xilesun/CS50x/assets/117184733/fe71cb34-1065-4ace-8404-70ea6f56e4f5">

请问这是什么原因？

**Comment by 2013xile** [comment]
看起来你应该先编译
